pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -301,6 +311,40 @@ private bool ValidateImportResultsArgs()
             return true;
         }
 
+        public static readonly Argument ARG_LIMIT_NOISE = new DocArgument(@"limit-noise", NUM_VALUE,
+            (c, p) => c.LimitNoise = p.ValueDouble);
+
+        public static readonly Argument ARG_CHROMATOGRAM_DISCARD = new DocArgument(@"chromatogram-discard",
```

Comment:
Too general a name. Should be --chromatograms-discard-unused or --chrom-discard-unused